.github/workflows/_base-migration.yml

```diff
@@ -12,7 +12,7 @@ on:
       python-version:
         required: false
         type: string
-        default: '3.10'
+        default: '3.13'
```

Comment:
That won't work as older code (read: checkouts of v14) is only compatibly up to 3.10.

Reply:
But this is also not the reason for the failure in the environment setup... :-)

Reply:
> That won't work as older code (read: checkouts of v14) is only compatibly up to 3.10.

Ah, makes sense!
Was wondering why one of the actions was stuck all the way at 310.

Reply:
The original error might be a bit more intricate and actually only be the symptom of a legitimate bug. We may want to investigate and fix this separately, though.